playground/nuxt.config.ts

```diff
@@ -0,0 +1,53 @@
+import FeedModule from '..'
+
+export default defineNuxtConfig({
+  modules: [
+    FeedModule
+  ],
```

Comment:
@manniL 

Would you help me with this type error?
This error is shown even after fresh install of Nuxt module starter.

By the way, it doesn't prevent the module from working.

<img width="1420" alt="Screen Shot 2022-12-15 at 3 15 47" src="https://user-images.githubusercontent.com/16436160/207674549-48a28c61-a592-4411-8ba9-800e8acd8597.png">

Reply:
I'll have a look ☺️

Reply:
📝 It seems that `feeds` module options is not correctly type-inferred due to this type error, but not sure.

Reply:
Sorry for the delay. Replaced with relative path, see https://github.com/nuxt/nuxt/issues/15750 for context